src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private final FileWriter fileWriter = new FileWriterImpl();
+    private final FileReader fileReader = new FileReaderImpl();
```

Comment:
initialize in a method with annotation Before or BeforeClass (depends on junit version)